documentation/src/main/asciidoc/topics/con_cache_encoding.adoc

```diff
@@ -0,0 +1,39 @@
+[id='encoding']
+= Cache encoding
+Encoding is the format, identified by a media type, that {brandname} uses to store entries (key/value pairs) in caches.
+
+[discrete]
+== Remote caches
+
+{brandname} Server stores entries in remote caches without any encoding information for keys and values, which means remote caches use the `application/unknown` media type by default.
```

Comment:
I'm still not 100% happy with this, it implies a causal relation. How about

```
{brandname} Server uses a single encoding to store all the keys (and a separate encoding for all the values) in a remote cache. The default encoding is `application/unknown`, meaning only the client knows the actual encoding.

Hot Rod and REST clients requests include a media type for keys and one for values.
```

Reply:
@danberindei Thanks, I think I see what you're getting at. I do have a question about your suggested text.

"and a separate encoding for all the values" - I read this to mean that `application/unknown` is for keys but there's something different for values. Is that the case? If so, then I think we'll need to clarify that further.

Could it be that the meaning is more along the lines of "the server uses the `application/unknown` media type to encode all keys and values, which have separate configurations"?

In this case maybe we should change back the ConfigurationBuilder to show the separate media type too?

Reply:
Better not to leak ```application/unknown``` in the documentation, otherwise uses will start using it, it's a deprecated media type meant originally for internal usage only

Reply:
thanks @gustavonalle we've removed the reference to `application/unknown`